AccountMasterKey.js

```diff
@@ -178,6 +179,30 @@ export class AccountMasterKey {
     const signer = await _signerFromSeed({seed});
     return new AccountMasterKey({accountId, signer, kmsService, kmsPlugin});
   }
+  /**
+   * Gets the master key from either the cache or a secret.
+   *
+   * @param {Object} options - Options for generating a master key.
+   * @param {string} options.accountId - An account id.
+   * @param {string} [options.secret = null]
+   * - A secret that is used to generate a key.
+   * @param {string} [options.plugin = 'ssm-v1']
+   * - The plugin type for the service.
+   * @param {Object} [options.Service = KmsService]
+   * - The service to use to get the key.
+   *
+   * @returns {AccountMasterKey} A key.
+   */
+  static getMasterKey(
```

Comment:
This naming convention doesn't match the rest of the class. Perhaps `fromCache` should just be updated to optionally accept a `secret` and to make passing the `kmsService` optional?

Reply:
or perhaps utils.js should just call explicitly on fromCache or fromSecret and not use this method at all?